ccmpred/sampling/__init__.py

```diff
@@ -156,7 +156,7 @@ def sample_with_mutation_rate(tree, nseq, seq0, x, mutation_rate):
 
     return msa_sampled, neff
 
-def sample_to_neff_increasingly(tree, nseq, target_neff, ncol, x, gibbs_steps):
+def sample_to_neff_increasingly(tree, nseq, target_neff, ncol, x, gibbs_steps, seq0 = None):
```

Comment:
If I am not mistaken, the logic you chose here does not preserve the previous logic in case of `seq0 != None`. I would have tried:

`root_seq=None`

Reply:
You are right !!!!!

Reply:
I added this to my main branch. Maybe I should create another branch? It is too late I think, but I wonder that is the proper way to update code with your comments?